pkgs/tools/graphics/qrcode/default.nix

```diff
@@ -13,6 +13,8 @@ stdenv.mkDerivation {
 
   NIX_CFLAGS_COMPILE = "-Wno-error=unused-result";
 
+  makeFlags = [ "CC=cc" ];
```

Comment:
```suggestion
  makeFlags = [ CC=${stdenv.cc.targetPrefix}cc ];
```

Reply:
Thanks @Mic92 ; I've incorporated the change and added you as co-author (if you don't mind).